### PR TITLE
add support to newer gcc (>4.9)

### PR DESCRIPTION
**Pull request by bobchennan**

### CODE REVIEW

File: tools/requirements_cuda91.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+chainer>=3.0.0
+cupy-cuda91>=2.0.0
```
Comment (sw005320) [inline tools/requirements_cuda91.txt]: In my setup, cupy>=2.0.0 is fine.

Comment (bobchennan) [reply tools/requirements_cuda91.txt]: let me try it

Comment (bobchennan) [reply tools/requirements_cuda91.txt]: Yes you are right

**Comment by sw005320** [comment]
Can you also add about this Makefile to README?

**Pull request merged**